WcaOnRails/app/assets/javascripts/registrations.js

```diff
@@ -21,7 +21,8 @@ onPage('registrations#edit_registrations', function() {
   showHideActions();
 
   $('button[value=delete-selected]').on("click", function(e) {
-    if(!confirm("Are you sure you want to delete the selected registrations?")) {
+    var $selectedRows = $registrationsTable.find("tr.selected");
+    if(!confirm("You are about to delete " + $selectedRows.length + " registrations.\nAre you sure you want to do it?")) {
```

Comment:
I thought about that, and decided to split the sentence to emphasize the number of registrations.

That's totally subjective though, so I'm open to change it if the explanation behind the long message does not convince you :p